examples/tests/sdk_tests/std/duration.w

```diff
@@ -2,10 +2,11 @@
 // seconds
 
 assert(12m.seconds == 12 * 60);
+assert(std.Duration.fromMinutes(12).seconds == 12 * 60);
```

Comment:
The `std` namespace is implicitly imported in wing. Do we have to specify the namespace `std.Duration.fromMinutes` or does `Duration.fromMinutes` works as well?

Reply:
I think we should get that to work without `std`. Let's see if someone from the team can come up with a direction how to achieve that

Reply:
yes the goal is to have this working without the `std`

Reply:
this is a Duration problem, not related to this 
opened 
https://github.com/winglang/wing/issues/2935